prime-router/src/main/kotlin/tokens/Scope.kt

```diff
@@ -7,14 +7,21 @@ import java.lang.IllegalArgumentException
 /**
  * A set of helper functions related to validating scopes.
  *
- * In ReportStream, a scope is of the form senderFullname.detailedScope,
- * that is, orgName.senderName.detailedScope
+ * In ReportStream, a scope is of the form senderOrReceiverFullname.detailedScope,
+ * that is, orgName.senderName.detailedScope or orgName.receiverName.detailedScope.
  * Example:  strac.default.report
  */
 class Scope {
     companion object : Logging {
+
+        /**
+         * Scope that represents prime admin access.
+         */
+        val primeAdminScope = "primeadmin.all.all"
```

Comment:
I'm introducing a new scope, to allow for PrimeAdmin access when accessing via Token auth.